<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-0KDP1L5" name="com.scp.invoice.I1Test" tests="8" failures="0" timestamp="7 Dec 2018 05:45:23 GMT" time="0.047" errors="1">
  <testcase name="Invoice14" time="0.018" classname="com.scp.invoice.I1Test"/>
  <testcase name="Invoice14" time="0.006" classname="com.scp.invoice.I1Test"/>
  <testcase name="Invoice14" time="0.003" classname="com.scp.invoice.I1Test"/>
  <testcase name="Invoice" time="0.000" classname="com.scp.invoice.I1Test">
    <error type="org.testng.TestNGException" message="
Method Invoice requires 3 parameters but 0 were supplied in the @Test annotation.">
      <![CDATA[org.testng.TestNGException: 
Method Invoice requires 3 parameters but 0 were supplied in the @Test annotation.
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:191)
	at org.testng.internal.Parameters.createParameters(Parameters.java:127)
	at org.testng.internal.Parameters.createParameters(Parameters.java:376)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:451)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1274)
	at org.testng.internal.Invoker.createParameters(Invoker.java:989)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1079)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- Invoice -->
  <testcase name="m1" time="0.005" classname="com.scp.invoice.I1Test"/>
  <testcase name="Invoice14" time="0.008" classname="com.scp.invoice.I1Test"/>
  <testcase name="Invoice15" time="0.004" classname="com.scp.invoice.I1Test"/>
  <testcase name="Invoice14" time="0.003" classname="com.scp.invoice.I1Test"/>
</testsuite> <!-- com.scp.invoice.I1Test -->
